spring-cloud-dataflow-server-kubernetes-autoconfig/src/main/java/org/springframework/cloud/dataflow/server/config/kubernetes/KubernetesDataFlowServerAutoConfiguration.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.dataflow.server.kubernetes;
+package org.springframework.cloud.dataflow.server.config.kubernetes;
```

Comment:
I think the conventions are <project>.<config> in terms of package name structure.

https://github.com/spring-cloud/spring-cloud-consul/tree/master/spring-cloud-consul-config/src/main/java/org/springframework/cloud/consul/config

https://github.com/spring-cloud/spring-cloud-vault-config/tree/master/spring-cloud-vault-config-consul/src/main/java/org/springframework/cloud/vault/config/consul